projects/angular/src/forms/datepicker/calendar.html

```diff
@@ -1,7 +1,7 @@
 <table class="calendar-table">
   <tr class="calendar-row weekdays">
-    <th *ngFor="let day of localeDays" class="calendar-cell weekday" role="heading" [attr.aria-label]="day.day">
-      {{day.narrow}}
+    <th *ngFor="let day of localeDays" class="calendar-cell weekday">
+      <span role="text" [attr.aria-label]="day.day">{{day.narrow}}</span>
```

Comment:
What is the reason to use role="text" here?
It is invalid ARIA role, that was proposed, but not made it to official spec and is sometimes used to workaround specific screenreader issues, which I do not see in this case.